.github/workflows/lint.yml

```diff
@@ -35,3 +35,17 @@ jobs:
         uses: pre-commit/action@v3.0.0
         with:
           extra_args: --all-files --hook-stage manual codespell || true
+
+  pep-lint:
+    name: Run pep-lint
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3"
+
+      - name: Run pep-lint
+        run: python pep-lint.py
```

Comment:
I guess this is okay as a temporary measure if you want to keep my old pygrep checks around while we make sure this script's results are consistent with them (namely in terms of false negatives, given we can already be quite confident of false positives if they don't occur on the entire existing PEPs dataset).

Of course, when it comes time to actually replace those checks, this should obviously be eliminated in favor of just adding them as a regular pre-commit check, so they don't consume an extra CI slot and run automatically or on-demand for users locally just like all the other checks, rather than needing to have their script invoked manually, plus it only runs if PEPs were actually modified. Like so:

```yaml
  - id: pep-lint
    name: Lint PEP headers
    entry: 'python pep-lint.py'
    language: system
    files: 'pep-\d{4}.(txt|rst)'
    pass_filenames: false
```

Additionally, if you have it take a list of files as arguments, you can take full advantage of pre-commit's built-in file discovery and selection features, to further save on runtime and only show errors from changed or selected PEPs by default.

Reply:
Added. How would the filenames-as-arguments work?